src/support/small_vector.h

```diff
@@ -49,7 +49,7 @@ namespace wasm {
 template<typename T, size_t N> class SmallVector {
   // fixed-space storage
   size_t usedFixed = 0;
-  std::array<T, N> fixed;
+  std::array<T, N> fixed{};
```

Comment:
Let's move this to a separate PR. Can we remove the pragmas that disable the warning, above, after it?